statsd/config.go

```diff
@@ -0,0 +1,44 @@
+package statsd
+
+import (
+	"runtime"
+	"time"
+
+	"github.com/Sirupsen/logrus"
+	"github.com/spf13/viper"
+)
+
+// Config encapsulates all of the parameters necessary for starting up
+// the statsd server.
+type Config struct {
+	Log              *logrus.Logger
+	Viper            *viper.Viper
```

Comment:
Similar comment about being a global singleton and not a lot of benefits doing that.